README.md

```diff
@@ -40,7 +40,7 @@ Here I assume that you want to install themes into directory
 
 2.  Clone this repository:
     
-        git clone git@github.com:dmitry-kabanov/tmux-statusline-themes.git
+        git clone git@github.com:trolley/tmux-statusline-themes.git
```

Comment:
Nice catch of outdated repo name (`kabanovdmitry`), thanks.
But I would prefer to keep `dmitry-kabanov` instead of `trolley`.

Reply:
Whoops, sorry! That was only meant for my fork until you had my PR merged. I'll fix this.

Reply:
Fixed, thank you :)